src/coalescenceml/artifacts/base_artifact.py

```diff
@@ -0,0 +1,61 @@
+from typing import Any, Dict
+
+from ml_metadata.proto import metadata_store_pb2
+from tfx.types.artifact import Artifact, Property, PropertyType
+
+from coalescenceml.artifacts.constants import (
+    DATATYPE_PROPERTY_KEY,
+    PRODUCER_PROPERTY_KEY,
+)
+
+
+DATATYPE_PROPERTY = Property(type=PropertyType.STRING)
+PRODUCER_PROPERTY = Property(type=PropertyType.STRING)
+
+
+class BaseArtifact(Artifact):
+    """ """
```

Comment:
Would be useful to explain what the difference between `BaseArtifact` and `BaseArtifactStore` is